src/utils/exec.js

```diff
@@ -23,3 +23,12 @@ export async function execFile (filePath, args) {
 export async function exec (command) {
     return await execPromise(command);
 }
+
+export async function execWinShellUtf8 (command) {
+    var setCodePageCmd     = `FOR /F  "tokens=2 delims=:" %i in ('chcp') do (chcp 65001`;
+
+    // NOTE: To avoid terminal errors, we need to restore original code page after command is executed.
```

Comment:
Please use the exact text I give you:
![image](https://cloud.githubusercontent.com/assets/11459924/12350242/e8be8446-bb86-11e5-894b-66f4780f3480.png)
```
To avoid terminal errors, we need to restore the original code page after the command is executed.
```